Dockerfile

```diff
@@ -18,16 +18,13 @@ RUN sed 's/#force_color_prompt=yes/force_color_prompt=yes/' /etc/skel/.bashrc >
 ADD .dockerfiles/.vimrc /root/
 
 WORKDIR /code/data-store
-ADD requirements-dev.txt .
+ADD requirements-dev.txt requirements.txt ./
```

Comment:
Ok cool, I've never used docker. :)

Just wanted to make sure it was intentional.